docker-compose.yml

```diff
@@ -55,3 +56,7 @@ services:
     ports:
       - 4567
 
+  stripe:
+    image: stripemock/stripe-mock:latest
+    ports:
+      - "12112:12112"
```

Comment:
maybe cfg value with default of these values?

Reply:
configuration for docker like an docker-compose `.env` file?